lib/rubocop/cop/lint/circular_argument_reference.rb

```diff
@@ -0,0 +1,24 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for circular argument references in keyword arguments.
+      #
+      # These emit an error on newer versions of Ruby.
```

Comment:
So, this should say this is a counterpart of a warning produced by MRI since Ruby 2.2.